### ISSUE TITLE
Save file dialog box does not appear on MacOS

### ISSUE THREAD

**Issue by bertrandboudaud** [samhocevar/portable-file-dialogs#65]
Thanks for this simple useful library.

We have integrated it into our software but on MacOS (only), the dialog box for saving file does not appear if we set a default name.

I have extracted the command which is:

``osascript -e "set ret to choose file name default location \"/User/berbou/sequence.csv\" with prompt \"Export Injections\""``

when executed on a terminal it gives this error:
``execution error: Can’t make "/User/berbou/sequence.csv" into type alias.``

I've been able to run that by replacing "location" by "name" in the command line:

``osascript -e "set ret to choose file name default name \"/User/berbou/sequence.csv\" with prompt \"Export Injections\""``

The dialog file then appears with the appropriate name:

![image](https://user-images.githubusercontent.com/1705849/150983717-adbeb20f-29be-4559-93cb-0e229ea0ab46.png)

  OS Monterey 12.1 (21C52)
  Model Name:	MacBook Pro
  Model Identifier:	MacBookPro14,1
  Processor Name:	Dual-Core Intel Core i5
  Processor Speed:	2,3 GHz
  Number of Processors:	1
  Total Number of Cores:	2
  L2 Cache (per Core):	256 KB
  L3 Cache:	4 MB
  Hyper-Threading Technology:	Enabled
  Memory:	8 GB
  System Firmware Version:	447.60.3.0.0
  OS Loader Version:	540.60.2~89
  SMC Version (system):	2.43f11

**Comment by EvanBalster** [comment]
I have also run into this issue.

There is a little problem with your workaround, though.  "with default name" causes the path to be interpreted as a filename rather than a location.  For my own workaround I simply disabled the default location for save dialogs.

There is some discussion here that suggests converting the path to HFS format and assigning it to an alias variable in a separate statement would work.  I verified this on the command line but have not yet attempted modifying the `portable-file-dialogs` in this way:
https://apple.stackexchange.com/questions/196493/applescript-choose-file-dialog-box-with-default-location-not-working
